TunnelVisionLabs.ReferenceAssemblyAnnotator/AnnotatorBuildTask.cs

```diff
@@ -50,7 +50,6 @@ public string OutputPath
             set;
         }
 
-        [Required]
         public string DisabledWarnings
```

Comment:
This should have been made nullable; apparently MSBuild either doesn't call the setter or uses `null` when the target's DisabledWarnings input evaluates to empty.

Reply:
Fixed by https://github.com/tunnelvisionlabs/ReferenceAssemblyAnnotator/pull/77